{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-6251",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://launchpad.support.sap.com/#/notes/2828558",
                    "name": "https://launchpad.support.sap.com/#/notes/2828558",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https://wiki.scn.sap.com/wiki/pages/viewpage.action?pageId=545396222",
                    "name": "https://wiki.scn.sap.com/wiki/pages/viewpage.action?pageId=545396222",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "Under certain conditions or error scenarios SAP Business Objects Business Intelligence Platform, version 4.2, allows an attacker to access information which would otherwise be restricted."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-05-12T18:15Z",
    "lastModifiedDate": "2020-05-12T18:31Z"
}